wire-ios-request-strategy/Sources/Synchronization/Decoding/EventDecoder+MLS.swift

```diff
@@ -0,0 +1,88 @@
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+extension EventDecoder {
+
+    func decryptMlsMessage(
+        from updateEvent: ZMUpdateEvent,
+        context: NSManagedObjectContext
+    ) -> ZMUpdateEvent? {
+        Logging.mls.info("decrypting mls message")
+
+        guard let mlsController = context.mlsController else {
+            Logging.mls.warn("failed to decrypt mls message: MLSController is missing")
```

Comment:
shouldn't be this like a fatalError, programming error?

Reply:
yes, added a fatal error